### PR TITLE
Introduce flows specific to objects.

### PR DESCRIPTION
**Pull request by nnDarshan**
This patch segregates flows specific to an object
under the "flows" of that object instead of the
global namespace.

tendrl-spec: segragate_object_specific_flows
tendrl-bug-id: Tendrl/gluster_integration#95
Signed-off-by: nnDarshan <darshan.n.2024@gmail.com>

### CODE REVIEW

**Comment by nnDarshan** [comment]
@shtripat @nthomas-redhat Please review

**Comment by nnDarshan** [comment]
@shtripat I feel there is no need for any code change for this needed in back end right? code change is needed only in API layer right ?

**Comment by shtripat** [comment]
@nnDarshan thats correct. @anivargi can you please comment on this?

File: tendrl/gluster_integration/manager/tendrl_definitions_gluster.py [hunk 1]
```diff
@@ -192,7 +143,56 @@
           run: tendrl.gluster_integration.objects.Volume.atoms.volume_stopped.VolumeStopped
           type: Check
           uuid: 242f6190-9b37-11e6-950d-a24fc0d9656c
-      attrs:
+      flows:
+        DeleteVolume:
+          atoms:
```
Comment (shtripat) [inline tendrl/gluster_integration/manager/tendrl_definitions_gluster.py]: As per latest refactoring `description` should be renamed as `help`

Comment (nnDarshan) [inline tendrl/gluster_integration/manager/tendrl_definitions_gluster.py]: Done

**Pull request merged**